app/models/host/base.rb

```diff
@@ -138,11 +138,6 @@ def self.import_host(hostname, certname = nil)
       host
     end
 
-    def import_facts(facts, source_proxy = nil)
```

Comment:
Looks like this one is still being used in Katello...

Reply:
Is there an open issue/PR to track this?